medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -162,7 +162,7 @@
                             </group>
                         </group>
                     </group>
-                    <group name="state" attrs="{'invisible': [('id','=',False)]}">
```

Comment:
The field needs to be defined in the view in order to be part of the domain. Are you sure this isn't also the issue with using `id`?

`<field name="id" invisible="1" />`